### ISSUE TITLE
Installing error

### ISSUE THREAD

**Issue by MislavSag** [a-hanf/mlr3automl#36]
There is an error when installing the new version of the package:
```
> devtools::install_github('a-hanf/mlr3automl')
Downloading GitHub repo a-hanf/mlr3automl@HEAD
√  checking for file 'C:\Users\Mislav\AppData\Local\Temp\Rtmp86dDna\remotes3efceab31af\a-hanf-mlr3automl-0a0d8c7/DESCRIPTION'
-  preparing 'mlr3automl':
√  checking DESCRIPTION meta-information ... 
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
-  building 'mlr3automl_0.0.0.9000.tar.gz'
   
Installing package into ‘C:/Users/Mislav/Documents/R/win-library/4.1’
(as ‘lib’ is unspecified)
* installing *source* package 'mlr3automl' ...
** using staged installation
** R
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
  converting help for package 'mlr3automl'
    finding HTML links ... done
    AutoML                                  html  
    finding level-2 HTML links ... done

    AutoMLBase                              html  
Error: C:/Users/Mislav/AppData/Local/Temp/RtmpOowfbs/R.INSTALL6c4c644b4c74/mlr3automl/man/AutoMLBase.Rd:271: Bad \link text
* removing 'C:/Users/Mislav/Documents/R/win-library/4.1/mlr3automl'
* restoring previous 'C:/Users/Mislav/Documents/R/win-library/4.1/mlr3automl'
Warning message:
In i.p(...) :
  installation of package ‘C:/Users/Mislav/AppData/Local/Temp/Rtmp86dDna/file3efc390535c3/mlr3automl_0.0.0.9000.tar.gz’ had non-zero exit status
```

**Comment by johnfrombluff** [comment]
Same on my system (Fedora Silverblue 34). Deleting the offending line results in a flawless compilation.

**Comment by a-hanf** [comment]
Ack, sorry for this. Will fix later today

**Comment by a-hanf** [comment]
Sorry, silly mistake (broken link in roxygen documentation).
fixed by #37

**Comment by claravetter** [comment]
Hi! I get have a similar error when installing the package using
`devtools::install_github('https://github.com/a-hanf/mlr3automl', dependencies = TRUE)`

**Error message:** 
```
ERROR: lazy loading failed for package ‘mlr3automl’
* removing ‘/opt/conda/lib/R/library/mlr3automl’
Warning message:
In i.p(...) :
  installation of package ‘/tmp/RtmpsKRulv/fileddc5bdd5429/mlr3automl_0.0.0.9000.tar.gz’ had non-zero exit status
```
  

```
> version
               _                           
platform       x86_64-conda-linux-gnu      
arch           x86_64                      
os             linux-gnu                   
system         x86_64, linux-gnu           
status                                     
major          4                           
minor          1.0                         
year           2021                        
month          05                          
day            18                          
svn rev        80317                       
language       R                           
version.string R version 4.1.0 (2021-05-18)
nickname       Camp Pontanezen          
```

Thank you very much!
